src/common/state/QueryOverTimeAttributes.xml

```diff
@@ -41,6 +41,9 @@
     <Field name="useCachedPts" label="useCachedPts" type="bool" access="protected">
       false
     </Field>
+    <Field name="canUseDirectDatabaseRoute" label="canUseDirectDatabaseRoute" type="bool" access="protected">
```

Comment:
Since this isn't a user-settable field, you should set the 'internal' attribute to true.

Reply:
Done